packages/itwin/grouping-mapping-widget/src/widget/components/QueryBuilder.ts

```diff
@@ -516,17 +512,16 @@ export class QueryBuilder {
       return "";
     }
 
-    let unionQuery = "";
+    let querySegments: string[] = [];
     for (const union of this.query.unions) {
       const baseClass = union.classes[0];
       const baseClassName = baseClass.className;
       const baseIdName = baseClass.isAspect
         ? `${baseClassName}.Element.id`
         : `${baseClassName}.ECInstanceId`;
 
-      let queryString = `SELECT ${baseIdName}${
-        baseClass.isAspect ? " ECInstanceId" : ""
-      } FROM ${baseClassName}`;
+        var selectSegment = `SELECT ${baseIdName}${baseClass.isAspect ? " ECInstanceId" : ""} FROM ${baseClassName}`;
```

Comment:
You should almost never use `var`. `selectSegment` should be const or should not even exist, since you immediately push it into the array.

Reply:
Changed to const